packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,377 @@
+## User Types API
```

Comment:
If this API is not yet in GA, we should add a lifecycle banner, e.g.:
``
<ApiLifecycle access="ea" />
```